### DISCUSSION TITLE
Feed post images width size is small and not visible entirely without clicking

### DISCUSSION THREAD

**Discussion by vpbgkt** [dailydotdev/daily#1253] [Feature Requests]
I appreciate the content, but I have an issue with the visibility of images in the feed. Due to their limited height, images are only fully visible when clicked. Please consider increasing the width and size of posts in the feed so that images can be viewed entirely without needing to click on them. Clicking on each image feels time-consuming and tedious.

**Comment by idoshamun** [comment]
Hi @vpbgkt,

Thanks for the suggestion. We will consider it :)
